### PR TITLE
Connected to api

### PR DESCRIPTION
**Pull request by tayor99**
I connect to the bright's api and edited the previous changes

### CODE REVIEW

File: tayo-blog/src/Pages/Home.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+              onClick={() => handleClick(blog.id)}
+            >
+              Read more
+            </Link>
+          </p>
+        </div>
+      </div>
+    );
+  });
+
```
Comment (Asylcreek) [inline tayo-blog/src/Pages/Home.js:45]: Why save this into a variable?

File: tayo-blog/src/Pages/Home.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+import axios from 'axios';
+
+export const Home = ({ setCurrentPost }) => {
+  const [blogs, setBlogs] = useState([]);
+
+  useEffect(() => {
+    const blogPost = async () => {
+      const { data } = await axios.get('https://blogpostapi1.herokuapp.com/');
+      setBlogs(data.data);
+    };
```
Comment (Asylcreek) [inline tayo-blog/src/Pages/Home.js:13]: No error handling?

File: tayo-blog/src/Pages/Newpost.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+export const Newpost = () => {
+  const [name, setName] = useState('');
+  const [title, setTitle] = useState('');
+  const [content, setContent] = useState('');
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+
+    axios.post(
```
Comment (Asylcreek) [inline tayo-blog/src/Pages/Newpost.js:12]: I believe you are supposed to `await` this

File: tayo-blog/src/Pages/Newpost.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+
+export const Newpost = () => {
+  const [name, setName] = useState('');
+  const [title, setTitle] = useState('');
+  const [content, setContent] = useState('');
```
Comment (Asylcreek) [inline tayo-blog/src/Pages/Newpost.js:7]: You can put each of these into one state item, like so
```jsx
const [newPost, setNewPost] = useState({name:"",title:"",content:""})
```

**Review by Asylcreek — CHANGES_REQUESTED** [review]

File: tayo-blog/src/Pages/Home.js (hunk 1 above)
Comment (tayor99) [reply tayo-blog/src/Pages/Home.js:45]: I wanted to make my code more readable

Comment (Asylcreek) [reply tayo-blog/src/Pages/Home.js:45]: Okay

**Pull request merged**